src/cli/commands/force/lightning/local/device/create.ts

```diff
@@ -199,79 +211,3 @@ export class Create extends Setup {
         );
     }
 }
-
```

Comment:
I'm confused... why are these extra requirement checks being removed? Similar to the `Preview` command, the `Create` command needs the base requirement checks + these extra checks.

Reply:
The classes below aren't used anywhere. I can put it back. Can you tell me though where they are referenced?

Reply:
They were used in lines 73-77 as `additionalRequirements`, which you removed:

```
        const extraReqs: Requirement[] = [
            new DeviceNameAvailableRequirement(this, this.logger),
            new ValidDeviceTypeRequirement(this, this.logger)
        ];
        this.addAdditionalRequirements(extraReqs);
```

Reply:
😬 
Thanks!